src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -510,6 +511,10 @@
 						"	set the fake max players count to (online players count + 1)",
 						"	set the shown icon to a random server icon out of {server-icons::*}")
 				.since("2.3");
+		if(Skript.classExists("org.bukkit.event.command.UnknownCommandEvent")) Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "unknown command")
+			.description("Called when a player sends a command that isn't registered")
+			.examples("on unknown command:")
+			.since("INSERT VERSION").requiredPlugins("Paper");
```

Comment:
Follow the other event structure... Move ``Skript.registerEvent`` onto the next line, same with ``.requiredPlugins``

Reply:
Will do, by the way could you add it to https://github.com/SkriptLang/Skript/blob/master/CODING_CONVENTIONS.md please?